### PR TITLE
isEmpty() method [List Interface]

### LINKED ISSUE #15: Implement : List Interface - boolean isEmpty()

### PR DESCRIPTION
**Pull request by jakansha2001**
Fixed Issue #15

### CODE REVIEW

File: Collection Interfaces/List Interface/boolean isEmpty()/isEmptyFunc.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import java.util.*;
```
Comment (jyotijauhari) [inline Collection Interfaces/List Interface/boolean isEmpty()/isEmptyFunc.java:7]: (in line 2-3)
add documentation(in form of comment explaining [ isEmpty() method ] in 2-3 lines.

File: Collection Interfaces/List Interface/boolean isEmpty()/isEmptyFunc.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import java.util.*; 
+
+public class isEmptyFunc { 
+	public static void main(String[] args)
```
Comment (jyotijauhari) [inline Collection Interfaces/List Interface/boolean isEmpty()/isEmptyFunc.java]: Please add space around operators at all places 
also space aound ( 
      space around [
      space around <

File: Collection Interfaces/List Interface/boolean isEmpty()/isEmptyFunc.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+		arr.add(16); 
+ 
+		answer = arr.isEmpty(); 
+		if (answer == true) 
+			System.out.println("The List is empty"); 
+		else
+			System.out.println("The List is not empty"); 
+	} 
+}
```
Comment (jyotijauhari) [inline Collection Interfaces/List Interface/boolean isEmpty()/isEmptyFunc.java:37]: add sample input and output in form of comments at last .

Comment (jyotijauhari) [reply Collection Interfaces/List Interface/boolean isEmpty()/isEmptyFunc.java:37]: mention issue number in description of pr.

eg:
Fixes #11 (issue_no)

File: Collection Interfaces/List Interface/boolean isEmpty()/isEmptyFunc.java (hunk 1 above)
Comment (jyotijauhari) [reply Collection Interfaces/List Interface/boolean isEmpty()/isEmptyFunc.java:7]: @jakansha2001  check once the comments i added in first line. From next time try include it too in the code.

**Review by jyotijauhari — APPROVED** [review]

**Comment by JyotiJauhari** [comment]
As it is method of list interface, so from next time add the file in
>>Scrabbling-Java-With-CF/Collection Interfaces/List Interface/

**Comment by jyotijauhari** [comment]
**_Congratulation_**  😇 , your first pr is merged . Keep Contributing , Keep learning.

File: Collection Interfaces/List Interface/boolean isEmpty()/isEmptyFunc.java (hunk 1 above)
Comment (jakansha2001) [reply Collection Interfaces/List Interface/boolean isEmpty()/isEmptyFunc.java:7]: @JyotiJauhari Thank you for this info. I will take care of this thing from now onwards.

**Pull request merged**